readme.md

```diff
@@ -8,9 +8,9 @@
 [![Downloads](https://img.shields.io/npm/dt/zustand.svg?style=flat&colorA=000000&colorB=000000)](https://www.npmjs.com/package/zustand)
 [![Discord Shield](https://img.shields.io/discord/740090768164651008?style=flat&colorA=000000&colorB=000000&label=discord&logo=discord&logoColor=ffffff)](https://discord.gg/poimandres)
 
-A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks, isn't boilerplatey or opinionated.
+A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks that isn't boilerplatey or opinionated.
```

Comment:
It's a little bit different semantically now. Previously, not being boilerplatey referred to Zustand. Now it's about API.

But I think this version is also alright.

Reply:
Sure then!

Reply:
I don't know but the original one is more intentional?

Reply:
> I don't know but the original one shows the intention more?

Let me repost it. Is there a grammar fix to preserve the original meaning?

Reply:
> I don't know but the original one shows the intention more?

Let me repost it. Is there a grammar fix to preserve the original meaning?

Reply:
Here are some examples, let's see which works best in this scenario.

- It has a comfortable API based on hooks that isn't boilerplatey or opinionated.
- Has a nice way of working that uses hooks and isn't too standard or opinionated.
-  Has a slick API built on hooks that's not stale or snarky.
- Has a nice API that's not too formal or opinionated.

Reply:
I think what's unopinionated is Zustand itself, not the API. @sewera Please help on this.

Reply:
Technically speaking, the only way of interacting with Zustand is through its API. That's why I approved that version as well. With that said, if the intention was specifically to mark Zustand as not opinionated or boilerplatey, let's go with the previous version.

Reply:
Now when I think of it, the previous version is probably better.

Reply:
Yeah, I think Zustand is an unopinionated library and its API allows less boilerplate.
Let's revert the change.